docs/usage/file-system.md

```diff
@@ -0,0 +1,48 @@
+(file-system)=
+
+# Dealing with Pyodide file system
+
+Pyodide supports file system through [Emscripten File System API](https://emscripten.org/docs/api_reference/Filesystem-API.html#filesystem-api).
+In JavaScript, Pyodide file system can be accessed through {any}`pyodide.FS`.
+
+**Example: Reading from the file system**
+
+```js
+pyodide.runPython(`
+  f = open("/hello.txt", "w")
+  f.write("hello world!")
+  f.close()
+`);
+
+let file = pyodide.FS.readFile("/hello.txt", { encoding: "utf-8" });
```

Comment:
No, encoding should be specified when reading text because [default encoding of `FS.readFile` is binary](https://emscripten.org/docs/api_reference/Filesystem-API.html#FS.readFile).

```js
console.log(pyodide.FS.readFile("/hello.txt", {encoding: "utf8"}))
hello world
console.log(pyodide.FS.readFile("/hello.txt"))
Uint8Array(11) [104, 101, 108, 108, 111, 32, 119, 111, 114, 108, 100, buffer: ArrayBuffer(11), byteLength: 11, byteOffset: 0, length: 11, Symbol(Symbol.toStringTag): 'Uint8Array']
```

But actually, it should be `utf8` not `utf-8`. It is a typo 🙄.